dima/react-movies/src/components/footer/footer.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import "./footer.css";
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="footer-container">
+        <h3 className="footer-logo">netflixroulette</h3>
```

Comment:
logo is usually a link. Please replace it with an "a" tag

Reply:
fixed